core/client/fs/src/main/java/alluxio/client/file/RetryHandlingFileSystemMasterClient.java

```diff
@@ -85,8 +85,8 @@ public RetryHandlingFileSystemMasterClient(MasterClientConfig conf) {
   }
 
   @Override
-  protected AlluxioService.Client getClient() {
-    return null;
+  protected AlluxioServiceType getRemoteServiceType() {
```

Comment:
Earlier, each thrift service was implementing a common AlluxioService service which was serving the service version to clients. Since they were related by inheritance, service implementation knew the service type and could query the version. Now with gRPC, and without service inheritance, we are exposing a dedicated service for serving service versions to clients, which means clients need to know what service type they are asking for. That's why getClient() is removed with getRemoteSErviceType() and in AbstractClient.checkVersions() we are making use of it to query the remote service version.